docs/application/dotnet/samples/tizen-samples.md

```diff
@@ -31,6 +31,18 @@
 			<p>SecureRepository is simple application that allows you to securely store data, certificates and cryptographic keys using C# Tizen.Security.SecureRepository API.</p>
 			</td>
 		</tr>
+		<tr>
+			<td>
+			<p><img alt="" height="360" src="media/w4voicememo.png" width="360" /></p>
+			</td>
+			<td>
+			<p><strong>VoiceMemo</strong></p>
+			<p>The Voice Memo sample application demonstrates how you can create an application using Xamarin.Forms API and Tizen FX APIs.</p>
```

Comment:
The Voice Memo sample application demonstrates how you can create an application using the Xamarin.Forms and Tizen FX APIs.</p>